[DOCS] Update Quickstart guide to be Cloud-compatible (#8036)
great-expectations/great_expectations: docs/docusaurus/docs/tutorials/quickstart/quickstart.md

# Great Expectations Quickstart

Use this quickstart to install GX, connect to sample data, build your first Expectation, validate your data, and review the validation results. This is a great place to start if you're new to GX and aren't sure if it's the right solution for you or your organization.

:::note Great Expectations Cloud

> This quickstart has been updated to work with both our open source and Cloud offerings of GX.

Reviewer: ```suggestion
You can use this quickstart with the open source Python version of GX or with Great Expectations Cloud.
```
Can you confirm this quickstart is compatible with Cloud? Also, is this option publicly available currently? If not, it can't be used with Cloud! :)

Author: Yup this is compatible with Cloud and will be publicly available once this merges (this is the last PR in the epic).

## Prerequisites

- Python versions 3.8 to 3.10. See [Python downloads](https://www.python.org/downloads/).

> - `pip`

Reviewer: ```suggestion
- pip
```
_pip_ is fine here. It's the terminology used in the pip documentation:

![image](https://github.com/great-expectations/great_expectations/assets/104205257/5ff83d34-1987-4124-b0b5-3f2079ac5179)

## Create Expectations

> - Run the following command to create two Expectations. The first Expectation uses domain knowledge (the `pickup_datetime` shouldn't be null), and the second Expectation uses [`auto=True`](../../guides/expectations/how_to_use_auto_initializing_expectations.md#using-autotrue) to detect a range of values in the `passenger_count` column.

Reviewer: ```suggestion
- Run the following command to create two Expectations:
```

    ```python name="tutorials/quickstart/quickstart.py create_expectation"
    ```

> 

Reviewer: ```suggestion
The first Expectation uses domain knowledge (the `pickup_datetime` shouldn't be null), and the second Expectation uses [`auto=True`](../../guides/expectations/how_to_use_auto_initializing_expectations.md#using-autotrue) to detect a range of values in the `passenger_count` column.
```

This is a step result and should appear _after_ the command.

> The first Expectation uses domain knowledge (the `pickup_datetime` shouldn't be null), and the second Expectation uses [`auto=True`](../../guides/expectations/how_to_use_auto_initializing_expectations.md#using-autotrue) to detect a range of values in the `passenger_count` column.

Reviewer: ```suggestion
The first Expectation uses domain knowledge (the `pickup_datetime` shouldn't be null), and the second Expectation uses [`auto=True`](../../guides/expectations/how_to_use_auto_initializing_expectations.md#using-autotrue) to detect a range of values in the `passenger_count` column.

```
